gensim/models/experimental/ReadMe.md

```diff
@@ -0,0 +1,6 @@
+# Similarity Learning Develop
```

Comment:
This file no needed, better to place it to python model docstring (head of file)